package.json

```diff
@@ -63,6 +63,9 @@
     "lint": "./node_modules/.bin/eslint lib",
     "lint:fix": "./node_modules/.bin/eslint --fix lib"
   },
+  "resolutions": {
+    "request/**/json-schema": "0.4.0"
+  },
```

Comment:
Fixes: https://github.com/advisories/GHSA-896r-f27r-55mw

Reply:
Maybe "^0.4.0" instead

Reply:
Does this actually work at all when we install suomifi-passport-saml in a project that uses npm, since resolutions seems to be a yarn specific thing? 🤔

Reply:
request is a devDependency so all good 👍